doc/setup/postgresql.md

```diff
@@ -18,8 +17,7 @@ If **geonetwork** is to be deployed, you need to create a dedicated user and sch
 ```
 createuser -SDRI geonetwork
 psql -d georchestra -c "ALTER USER geonetwork WITH PASSWORD 'www-data';"
-psql -d georchestra -c 'CREATE SCHEMA geonetwork;'
-psql -d georchestra -c 'GRANT ALL PRIVILEGES ON SCHEMA geonetwork TO "geonetwork";'
+psql -d georchestra -c 'CREATE SCHEMA AUTHORIZATION geonetwork;'
```

Comment:
Indeed, I did not know this was possible ... from the [pg doc](http://www.postgresql.org/docs/9.1/static/sql-createschema.html)
```
Create a schema for user joe; the schema will also be named joe:
CREATE SCHEMA AUTHORIZATION joe;
```
Though I don't find it very clear what it does ... I'd prefer 
```
CREATE SCHEMA geonetwork AUTHORIZATION geonetwork;
```
Anyway, ok for me !